deploy/roles/rhui_nodes/tasks/main.yml

```diff
@@ -43,12 +43,16 @@
 
 
 - name: perform repolist
-  shell: "yum repolist --disableplugin=* --disablerepo=* --enablerepo=local-rhui3 -v | grep 'Repo-pkgs' | cut -d: -f 2"
+  yum:
```

Comment:
hmmmm. I think, this will not work for RHEL8. And at some point 'yum' will be deprecated on Fedora. Is it possible to replace it with a `package` module?

Reply:
The package module unfortunately doesn't offer any usable parameters to examine repository information; see http://docs.ansible.com/ansible/latest/modules/package_module.html. It's only meant to install/upgrade/remove packages.

It's not clear yet how RHEL 8 will be handled, but if the yum module doesn't work there, we'll use the dnf module in a conditionally executed task. Or we could consider removing this step completely as it's just an additional check; if the local-rhui3 repo isn't working, the installation of rhui-installer and everything else will fail anyway.

Reply:
@RadekBiba and if the installation fails without local-rhui3 repo, will there be an informative msg that repo is missing? If so, I would remove this repo check step.

Reply:
I haven't checked that, but I don't think there will be any specific message about a missing repo. The installation will fail because rhui-installer won't be found in any of the enabled repos. I'll check that.

Reply:
Hm, ok.. I will merge it now. If times when `yum` stops to work come, it can be fixed then.